modules/rostests/apitests/browseui/SHExplorerParseCmdLine.c

```diff
@@ -22,21 +23,7 @@
 #define TEST_PATHA "C:\\SHExplorerParseCmdLine.test"
 #define TEST_PATHW WIDEN(TEST_PATHA)
 
-#define PADDING_SIZE 0x100
-
-typedef struct _EXPLORER_INFO
-{
-    PWSTR FileName;
-    PIDLIST_ABSOLUTE pidl;
-    DWORD dwFlags;
-    ULONG Unknown1[5];
-    PIDLIST_ABSOLUTE pidlRoot;
-    CLSID clsid;
-    GUID guidInproc;
-    ULONG Padding[PADDING_SIZE];
-} EXPLORER_INFO, *PEXPLORER_INFO;
-
-UINT_PTR (WINAPI *SHExplorerParseCmdLine)(_Out_ PEXPLORER_INFO Info);
```

Comment:
Why would you remove this?
It is useful that the test has a 'known good' struct, so it can be compared against the header one?
Otherwise the test depends on the header / struct it is testing...

Reply:
> Why would you remove this?

Is it not what Thomas and Giannis suggested on Jira?

> compared against the header one

Well, the test file is marked as testing the function (only), not investigating the struct (too).
My point of view would be the opposite: do test (with) the actual struct, not a (different) local one...

Reply:
yes (sure) why (not)

Reply:
is (this) struct (documented) somewhere?
Otherwise (we) lose (a) major (source) of (documentation),
and (in) a (few) years (people) will ask how we came by this struct.

Reply:
> is (this) struct (documented) somewhere?

Not that I know of.

> Otherwise (we) lose (a) major (source) of (documentation),

Which documentation?
Sorry, I don't understand how having two different structs helps anything.

> and (in) a (few) years (people) will ask how we came by this struct.

Is it not already the case, somehow?
What is your proposal, then?